pkg/exec/run.go

```diff
@@ -135,7 +135,7 @@ func PrepareCommand(
 	ctx context.Context,
 	component string,
 	version v0manifest.Version,
-	binPath, tag, wd string,
+	binPath, tag, instanceDir string, wd string,
```

Comment:
Seems we don't need `wd` anymore.

Reply:
playground is using this `PrepareCommand` too and set the wd as ~/.data/{playground}/{instance}